google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITQueryWatchTest.java

```diff
@@ -384,6 +384,36 @@ public void limitToLast() throws Exception {
     listenerAssertions.addedIdsIsAnyOf(emptyList(), asList("doc2", "doc3"));
   }
 
+  @Test
+  public void shutdownNowRejectsListener() throws Exception {
```

Comment:
nit: rejects adding a new listener?

Reply:
This tests verifies rejecting an existing listener. I renamed it to shutdownNowTerminatesActiveListener.